src/webui/modules/home/index.js

```diff
@@ -31,8 +31,8 @@ export default class Home extends React.Component {
     this.loadPackages();
   }
 
-
-  componentDidUpdate(prevProps, prevState) { // eslint-disable-line no-unused-vars
+  // eslint-disable-next-line no-unused-vars
```

Comment:
IS this necessary? Looks like both vars are used

Reply:
Nice catch. I'll change.